tests/e2e/gc_rwx_deployments.go

```diff
@@ -244,48 +244,45 @@ var _ = ginkgo.Describe("[rwm-csi-tkg] File Volume Provision with Deployments",
 				crdCNSFileAccessConfig, crdVersion, crdGroup, true)
 		}
 
-		ginkgo.By("Scale down deployment to 2 replica")
+		ginkgo.By("Scale down deployment to 1 replica")
 		dep, err = client.AppsV1().Deployments(namespace).Get(ctx, dep.Name, metav1.GetOptions{})
 		gomega.Expect(err).NotTo(gomega.HaveOccurred())
 
 		rep := dep.Spec.Replicas
-		*rep = 2
+		*rep = 1
 		dep.Spec.Replicas = rep
 		dep, err = client.AppsV1().Deployments(namespace).Update(ctx, dep, metav1.UpdateOptions{})
 		gomega.Expect(err).NotTo(gomega.HaveOccurred())
 		time.Sleep(sleepTimeOut * time.Second)
 
-		pods2, err := fdep.GetPodsForDeployment(client, dep)
+		_, err = fdep.GetPodsForDeployment(client, dep)
 		gomega.Expect(err).NotTo(gomega.HaveOccurred())
 
 		list_of_pods, err := fpod.GetPodsInNamespace(client, namespace, ignoreLabels)
 		gomega.Expect(err).NotTo(gomega.HaveOccurred())
 
-		num_of_pods := len(list_of_pods)
-		for i := 0; i < num_of_pods; i++ {
-			framework.Logf("Pod which are running after scale-down to 2 is %s", list_of_pods[i].Name)
+		if list_of_pods[0].Name == pods.Items[0].Name {
```

Comment:
can you put a comment here on what we are checking here

Reply:
Added a log message